src/transformers/data/data_collator.py

```diff
@@ -2028,7 +2038,14 @@ def __call__(self, features, return_tensors=None, separator_id=None):
             else:
                 batch["labels"] += [separator_id] + input_ids[1:]
             if self.return_position_ids:
-                batch["position_ids"] += list(range(len(input_ids)))
+                if is_sequence_length_provided:
```

Comment:
```suggestion
                if "sequence_length" in features[0]:
```

Reply:
same logic applies to `is_labels_provided` variable -- i find current implementation cleaner?